engine/main.cc

```diff
@@ -150,6 +150,8 @@ void RunServer(std::optional<int> port, bool ignore_cout) {
   LOG_INFO << "Please load your model";
 #ifndef _WIN32
   drogon::app().enableReusePort();
+#else
+  drogon::app().enableDateHeader(false);
```

Comment:
just wonder, if we send back to client RFC1123 date, then why we need to turn off DateHeader?

Reply:
The date field is not unique in drogon header. If we do not turn off DateHeader we will have 2 date fields.